### PR TITLE
Pod lifecycle.md

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
> 
> For pull requests on Chinese localization, set Base Branch to release-1.15
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: HostAliases
+id: HostAliases
+date: 2019-01-31
+full_link: /docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-core
+short_description: >
+  A HostAliases is a mapping between the IP address and hostname to be injected into a Pod's hosts file.
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/host-aliases.md]: translate this too.

File: content/zh/docs/reference/glossary/pod-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+id: pod-lifecycle
+date: 2019-02-17
+full-link: /docs/concepts/workloads/pods/pod-lifecycle/
+related:
+ - pod
+ - container
+tags:
+ - fundamental
+short_description: >
+  A high-level summary of what phase the Pod is in within its lifecyle.
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/pod-lifecycle.md]: translate this pls.

File: content/zh/docs/reference/glossary/pod-lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: Pod Lifecycle
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/pod-lifecycle.md]: Pod 生命周期

File: content/zh/docs/reference/glossary/pod-lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ - fundamental
+short_description: >
+  A high-level summary of what phase the Pod is in within its lifecyle.
+ 
+---
+<!--
+ A high-level summary of what phase the Pod is in within its lifecyle.
+-->
+
+关于 Pod 在其生命周期内处于什么阶段的高级摘要。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/pod-lifecycle.md]: high-level 不是高级的意思，请斟酌

File: content/zh/docs/reference/glossary/pod-lifecycle.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+<!--more--> 
+
+<!--
+The [Pod Lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/) is a high level summary of where a Pod is in its lifecyle.  A Pod’s `status` field is a [PodStatus](/docs/reference/generated/kubernetes-api/v1.13/#podstatus-v1-core) object, which has a `phase` field that displays one of the following phases: Running, Pending, Succeeded, Failed, Unknown, Completed, or CrashLoopBackOff.
+-->
+
+[Pod Lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/)是 Pod 在其生命周期中的高级摘要。 Pod 的 'status' 字段是 [PodStatus](/docs/reference/generated/kubernetes-api/v1.13/#podstatus-v1-core)对象，它有一个 'phase' 字段，显示以下阶段之一 ：运行，挂起，成功，失败，未知，已完成或回滚。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/pod-lifecycle.md]: Running ... 等，这些阶段的名字不要翻译

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+-->
+
+---
+title: HostAliases
+id: HostAliases
+date: 2019-01-31
+full_link: /docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-core
+short_description: >
+  HostAliases 是 IP 地址和要注入到 Pod 主机文件中的主机名之间的映射
+  A HostAliases is a mapping between the IP address and hostname to be injected into a Pod's hosts file.
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/host-aliases.md]: line 23 should be removed

Comment (tengqm) [reply content/zh/docs/reference/glossary/host-aliases.md]: move line 16-28 to the head of the file.

File: content/zh/docs/reference/glossary/pod-lifecycle.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+关于 Pod 在其生命周期内处于什么阶段的概括总结。
+
+<!--more--> 
+
+<!--
+The [Pod Lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/) is a high level summary of where a Pod is in its lifecyle.  A Pod’s `status` field is a [PodStatus](/docs/reference/generated/kubernetes-api/v1.13/#podstatus-v1-core) object, which has a `phase` field that displays one of the following phases: Running, Pending, Succeeded, Failed, Unknown, Completed, or CrashLoopBackOff.
+-->
+
+[Pod Lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/)是 Pod 在其生命周期中的概括总结。 Pod 的 'status' 字段是 [PodStatus](/docs/reference/generated/kubernetes-api/v1.13/#podstatus-v1-core)对象，它有一个 'phase' 字段，显示以下阶段之一 ：Running, Pending, Succeeded, Failed, Unknown, Completed, or CrashLoopBackOff。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/pod-lifecycle.md]:
```suggestion
[Pod Lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/)是 Pod 在其生命周期中的概括总结。 Pod 的 'status' 字段是 [PodStatus](/docs/reference/generated/kubernetes-api/v1.13/#podstatus-v1-core)对象，它有一个 'phase' 字段，显示以下阶段之一 ：Running、Pending、Succeeded、Failed、Unknown、Completed 或者 CrashLoopBackOff。
```

**Comment by markthink** [comment]
@congfairy  冲突需要解决一下，可尝试本地构建测试一下。

**Pull request merged**